## Applied statistics papers
#### Zaitlen Lab
#### Summer 2018


### References

1. Peters, L. A. et al. "A functional genomics predictive network model identifies regulators of inflammatory bowel disease." *Nature Genetics* 49, 1437–1449 (2017).
2. Liley, J., Todd, J. A. & Wallace, C. "A method for identifying genetic heterogeneity within phenotypically defined disease subgroups." *Nature Genetics* 49, 310–316 (2017).
3. Forsberg, S. K. G., Bloom, J. S., Sadhu, M. J., Kruglyak, L. & Carlborg, Ö. "Accounting for genetic interactions improves modeling of individual quantitative trait phenotypes in yeast." *Nature Genetics* 49, 497–503 (2017).
4. Consortium, T. B. et al. "Analysis of shared heritability in common disorders of the brain." *Science* 360, eaap8757 (2018).
5. Lan, X. & Pritchard, J. K. "Coregulation of tandem duplicate genes slows evolution of subfunctionalization in mammals." *Science* 352, 1009–1013 (2016).
6. Knowles, D. A. et al. "Determining the genetic basis of anthracycline-cardiotoxicity by molecular response QTL mapping in induced cardiomyocytes." *eLife* (2018). doi:10.7554/eLife.33480
7. Wen, L. et al. "Genome-scale detection of hypermethylated CpG islands in circulating cell-free DNA of hepatocellular carcinoma patients." *Cell Res* 25, 1250–1264 (2015).
8. Hickey, J. M. et al. "Genomic prediction unifies animal and plant breeding programs to form platforms for biological discovery." *Nature Genetics* 49, 1297–1303 (2017).
9. Zhao, W. et al. "Identification of new susceptibility loci for type 2 diabetes and shared etiological pathways with coronary heart disease." *Nature Genetics* 49, 1450–1457 (2017).
10. Barbeira, A. N. et al. "Integrating Predicted Transcriptome From Multiple Tissues Improves Association Detection." (2018). doi:10.1101/292649
11. Balazard, F. et al. "Interactions and collider bias in case-only gene-environment data" *bioRxiv*
12. Gymrek, M., Willems, T., Reich, D. & Erlich, Y. "Interpreting short tandem repeat variations in humans using mutational constraint." *Nature Genetics* 49, 1495–1501 (2017).
13. Zhu, H., Shen, D., Peng, X. & Liu, L. Y. "MWPCR: Multiscale Weighted Principal Component Regression for High-Dimensional Prediction." *Journal of the American Statistical Association* 112, 1009–1021 (2017).
14. Domingo, J., Diss, G. & Lehner, B. Pairwise and higher-order genetic interactions during the evolution of a tRNA. Nature 558, 117–121 (2018).
15. Chowell, D. et al. "Patient HLA class I genotype influences cancer response to checkpoint blockade immunotherapy." *Science* eaao4572 (2017). doi:10.1126/science.aao4572
16. Bastarache, L. et al. "Phenotype risk scores identify patients with unrecognized Mendelian disease patterns." *Science* 359, 1233–1239 (2018).
17. Zhao, S. D., Cai, T. T., Cappola, T. P., Margulies, K. B. & Li, H. S"parse Simultaneous Signal Detection for Identifying Genetically Controlled Disease Genes." *Journal of the American Statistical Association* 112, 1032–1046 (2017).
18. Park, S., Supek, F. & Lehner, B. "Systematic discovery of germline cancer predisposition genes through the identification of somatic second hits." *Nature Communications* 9, 2601 (2018).
19. Hore, V. et al. "Tensor decomposition for multiple-tissue gene expression experiments." *Nature* Genetics 48, 1094–1100 (2016).
20. Greene, C. S. et al. "Understanding multicellular function and disease with human tissue-specific networks." *Nature Genetics* 47, 569–576 (2015).
21. Dey, K. K. et al, "Visualizing the structure of RNA-seq expression data using grade of membership models. " *PLOS Genetics* 
